src/pages/events/summit-event-list-page/components/GridDisplayControl/GridDisplayControl.jsx

```
@@ -0,0 +1,105 @@
+  Menu,
+  MenuItem,
+  Box,
+  Button,
+  Typography,
+  Checkbox
+} from "@mui/material";
+import DisplayButton from "./components/DisplayButton";
+import { saveColumns } from "./actions/display-control-actions";
+
+
+const GridDisplayControl = ({ id, columns, selected, saveColumns }) => {
```

ESLint

[error] 'saveColumns' is defined but never used.

(no-unused-vars)